OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -28,9 +29,11 @@ public class FindResources : Activity
 		readonly IPathFinder pathFinder;
 		readonly DomainIndex domainIndex;
 
-		CPos? avoidCell;
+		CPos? orderLocation;
+		Actor deliverActor;
+		bool deliveredLoad;
 
-		public FindResources(Actor self)
+		public FindResources(Actor self, Actor deliverActor = null)
```

Comment:
What do you think about renaming this to `FindAndDeliverResources`, `AutoHarvest`, `HarvestLoop` or something else that encompasses its newly expanded role?

Reply:
Agreed.